Server/docker-compose.yaml

```diff
@@ -45,12 +56,18 @@ services:
     ports:
       - 4004:4004
       - 8800:8800
+    secrets:
+     - source: validator_logging
+       target: /etc/sawtooth/log_config.toml
+     - source: telegraf_logging
+       target: /etc/telegraf/telegraf.conf
     command: |
         bash -c 'if [[ ! -f /etc/sawtooth/keys/validator.pub ]]
             then 
                 echo "First run"
                 sawadm keygen
             fi
+            (telegraf &) && \
```

Comment:
What is `telegraf` and why do we start it only for validator but doesn't seem to be started for the other containers ?